docs/reference/components/sources/kubernetes_logs.cue

```diff
@@ -359,6 +359,20 @@ components: sources: kubernetes_logs: {
 				"""
 		}
 
+		metrics: {
+			title: "Metrics"
+			body: """
+				Our Helm chart deployments provide QoL around setup and maintenance of
+				metrics pipelines in Kubernetes. Each of the Helm charts provide an internal_metrics
```

Comment:
should `internal_metrics` and `prometheus` and `host_metrics` be code-formatted consistently?

Reply:
yeah absolutely!